main.go

```diff
@@ -72,6 +73,26 @@ func runCommand() error {
 		return nil
 	}
 
+	// read tsa and include-certs from gitconfig
+	path, err := os.Getwd()
+	if err == nil {
+		repo, err := git.OpenRepository(path)
+		if err == nil {
```

Comment:
Same question about failing open. The idiom used throughout is `err != nil` and returning an error. Is the idea here that we may not be in a repo and hence shouldn't fail? I'm not sure if the libgit2 bindings have helpers for this..but if we determine we aren't in a repo and/or there is no `.gitconfig` for that repo, wouldn't the most idiomatic thing be to check the user global and then system `.gitconfig` instead?